src/main/java/com/githubai/bot/AppController.java

```diff
@@ -13,6 +14,12 @@ public String index() {
         return "ok";
     }
 
+    // TODO: Unify this method with postHandler since both return the same response structure.
```

Comment:
Please create an issue on github add put the ID of the commit in the comment e.g: // TODO(#10): ....